ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -178,8 +178,15 @@
         provider-id (:provider params)
         native-id (:native-id route-params)
         concept-type (keyword (:concept-type route-params))
-        query-params (assoc {} :provider-id provider-id :native-id native-id)]
-    (mdb2/find-concepts request-context query-params concept-type {:raw? true})))
+        query-params (assoc {} :provider-id provider-id :native-id native-id)
+        found-concepts (mdb2/find-concepts request-context query-params concept-type {:raw? true})
+        ;; in memory database is inserting these values, where as the sql is not
```

Comment:
@eereiter please tell me if there is a better place to do this kind of thing.

Reply:
it was suggested in slack that we revert the code, relax the test, and update an existing ticket to correct this issue.